db/migrate/20241022153307_add_trackable_to_billing_usage_tracker.rb

```diff
@@ -0,0 +1,7 @@
+class AddTrackableToBillingUsageTracker < ActiveRecord::Migration[7.1]
+  def change
+    add_column :billing_usage_trackers, :trackable_id, :bigint
+    add_column :billing_usage_trackers, :trackable_type, :string, default: "Team"
```

Comment:
Instead of defaulting to `"Team"` should we maybe default to `BulletTrain::Billing::Usage.parent_association` in case someone has overridden that association?